hail/src/main/c/Makefile

```diff
@@ -1,11 +1,12 @@
-.PHONY: top all clean debug prebuilt test
-
-top: all
+MAKEFLAGS += --no-builtin-rules
+.PHONY: all clean debug prebuilt test
+.SUFFIXES:
+.DEFAULT_GOAL: all
```

Comment:
I know this is wierd, but `.DEFAULT_GOAL` is defined as a variable, not a target.
```make
.DEFAULT_GOAL := all
```